sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2775,6 +2775,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val LEGACY_CENTRAL_MOMENT_AGG_BEHAVIOR =
+    buildConf("spark.sql.legacy.centralMomentAgg.enabled")
+      .internal()
+      .doc("When set to true, stddev_samp and var_samp will return Double.NaN, " +
+        "if applied to a set with a single element. Otherwise, will return 0.0, " +
+        "which is aligned with TPCDS standard.")
```

Comment:
I think we don't need to describe  `which is aligned with TPCDS standard.` here for user documents.